tests/common/reboot.py

```diff
@@ -344,7 +344,9 @@ def get_reboot_cause(dut):
     # is such like "User issued 'xxx' command [User: admin, Time: Sun Aug  4 06:43:19 PM UTC 2024]"
     # So, use the above pattern to get real reboot cause
     if dut.facts["asic_type"] == "vs":
-        cause = re.search("User issued '(.*)' command", cause).groups()[0]
+        match = re.search("User issued '(.*)' command", cause)
```

Comment:
@yutongzhang-microsoft since KVM prints something like this for "show reboot-cause" on the first boot -

```
Unknown (First boot of SONiC version master.597250-dc3c0451b)
```

The above regex `User issued ...` won't match the above string. Won't you need one more regex `Unknown (\(.*\))` to match the above?

Reply:
Because, the pattern `User issued '(.*)' command` will get the real reboot reason in the position of `(.*)`, such as warm-reboot, fast-reboot. But if we get the string `Unknown (First boot of SONiC version master.597250-dc3c0451b)`, we can't match the cause of dict `reboot_ctrl_dict` in this file. So I think just return `REBOOT_TYPE_UNKNOWN` makes sence.

Reply:
Thank you I see what you mean. LGTM